lib/dnsimple.js

```diff
@@ -22,7 +22,7 @@ const services = {
 Dnsimple.services = services;
 Dnsimple.DEFAULT_TIMEOUT = require('http').createServer().timeout;
 Dnsimple.DEFAULT_BASE_URL = 'https://api.dnsimple.com';
-Dnsimple.DEFAULT_USER_AGENT = 'dnsimple-node/v2';
+Dnsimple.DEFAULT_USER_AGENT = 'dnsimple-node/2.4.0';
```

Comment:
I know. That's my concern too.

Is there a way we could define the version in a single place and use it from the `package.json` somehow?